WordPress/src/main/res/values/roles.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="roles" translatable="false">
+        <item>
+            Administrator
+        </item>
+        <item>
+            Editor
+        </item>
+        <item>
+            Author
+        </item>
+        <item>
+            Contributor
+        </item>
+    </string-array>
```

Comment:
Noticed that previously, strings.xml was defining a string constant for the Super Admin as well. Not sure if it's missing from this array by design or not. Cheers!